narwhals/dtypes.py

```diff
@@ -421,6 +421,24 @@ class Duration(TemporalType):
 
     Notes:
         Adapted from [Polars implementation](https://github.com/pola-rs/polars/blob/py-1.7.1/py-polars/polars/datatypes/classes.py#L460-L502)
+
+    Examples:
+        >>> import pandas as pd
+        >>> import polars as pl
+        >>> import pyarrow as pa
+        >>> import narwhals as nw
+        >>> data = [1000, 2000, 3000]  # 1s, 2s, 3s
+        >>> ser_pd = pd.to_timedelta(pd.Series(data), unit="ms")
```

Comment:
you'll need `.astype('timedelta64[ms]')` to cast the dtype

would it work to create these from lists of `timedelta`? where you do `from datetime import timedelta`?